bundles/statistics/statsgrid2016/components/SeriesControl.js

```diff
@@ -0,0 +1,293 @@
+Oskari.clazz.define('Oskari.statistics.statsgrid.SeriesControl', function (sandbox, locale) {
+    this.sb = sandbox;
+    this.loc = locale;
+    this.log = Oskari.log('Oskari.statistics.statsgrid.SeriesControl');
+    this.service = this.sb.getService('Oskari.statistics.statsgrid.StatisticsService');
+    this.seriesService = this.service.getSeriesService();
+    this.__templates = {
+        main: _.template(
+            '<div class="statsgrid-series-control-container">' +
+                '<svg class="line-svg">' +
+                    '<g class="axis"></g>' +
+                    '<g class="bg"><rect x="-10" y="-10" width="10" height="10"/></g>' +
+                    '<g class="drag-handle" cursor="ew-resize">' +
+                        '<rect x="-9" y="-9" width="18" height="18" fill-opacity="0"/>' +
+                        '<circle cx="0" cy="0" r="4"/>' +
+                    '</g>' +
+                '</svg>' +
+                '<div class="value-controls">' +
+                '</div>' +
+            '</div>')
+    };
+    this._lineWidth = 500;
+    this._minWidth = 290;
+    this._uiState = {
+        values: [],
+        currentSeriesIndex: undefined
+    };
+    this._bindToEvents();
+}, {
+    __speedOptions: [
+        { key: 'fast', value: 1000 },
+        { key: 'normal', value: 2000 },
+        { key: 'slow', value: 3000 }
+    ],
+    /**
+     * @method _generateSelectOptions Generate localized options for <select> dropdowns
+     * @private
+     * @param  {String} prefix localization path prefix
+     * @param  {Object[]} options key, value to localize
+     * @return {Object[]} key, value that has been localized
+     */
+    _generateSelectOptions: function (prefix, options) {
+        var me = this;
+        return options.map(function (e) {
+            return {
+                title: me.loc(prefix + e.key),
+                value: e.value
+            };
+        });
+    },
+    /**
+     * @method _initControls Set up previous, play/pause and next buttons, speed menu and value display
+     * @private
+     */
+    _initControls: function () {
+        var me = this;
+        var controlPanel = me._element.find('.value-controls');
+
+        var template = jQuery(
```

Comment:
There's a __templates variable in the constructor. This template should be defined in it.